### ISSUE TITLE
Fails to parse bookings

### ISSUE THREAD

**Issue by williamleven** [sidusIO/BooGroCha#20]
create one booking and try to list bookings

```
panic: parsing failure

goroutine 1 [running]:
sidus.io/boogrocha/internal/cli/commands.runList(0xc00015a500, 0xd81f48, 0x0, 0x0, 0x960f60)
        /home/gurgy/.cache/yay/boogrocha/src/boogrocha-r43.6342c69/internal/cli/commands/list.go:30 +0x6d7
sidus.io/boogrocha/internal/cli/commands.ListCmd.func1(0xc00015a500, 0xd81f48, 0x0, 0x0)
        /home/gurgy/.cache/yay/boogrocha/src/boogrocha-r43.6342c69/internal/cli/commands/list.go:18 +0x52
github.com/spf13/cobra.(*Command).execute(0xc00015a500, 0xd81f48, 0x0, 0x0, 0xc00015a500, 0xd81f48)
        /home/gurgy/go/pkg/mod/github.com/spf13/cobra@v0.0.4-0.20190321000552-67fc4837d267/command.go:766 +0x2ae
github.com/spf13/cobra.(*Command).ExecuteC(0xd5c760, 0x847ea3, 0xc000131f88, 0xc000096058)
        /home/gurgy/go/pkg/mod/github.com/spf13/cobra@v0.0.4-0.20190321000552-67fc4837d267/command.go:850 +0x2fc
github.com/spf13/cobra.(*Command).Execute(...)
        /home/gurgy/go/pkg/mod/github.com/spf13/cobra@v0.0.4-0.20190321000552-67fc4837d267/command.go:800
main.main()
        /home/gurgy/.cache/yay/boogrocha/src/boogrocha-r43.6342c69/cmd/bgc/main.go:9 +0x2e
```

**Comment by HerrNaN** [comment]
The Time Edit booking system seems to have changed the layout of the "My Bookings" page a bit causing the parsing of the said page to fail

**Comment by HerrNaN** [comment]
So apparently this bug wasn't solved by #22. This bug, however now only manifests in production and not run locally :/

**Comment by HerrNaN** [comment]
Reopened because fix didn't seem to work in production but it did, just didn't check the right one
